### ISSUE TITLE
doc issue

### ISSUE THREAD

**Issue by knabbers** [labthings/picamerax#17]
"The analog gain cannot be directly controlled in picamerax, but various attributes can be used to “influence” it."

I think that's no longer true?

**Comment by jtc42** [comment]
@rwb27 this _is_ still true right? When we "set" the analog gain we're just putting a nice interface on these tricks, I think.

**Comment by rwb27** [comment]
No, when we set the analog gain we are actually setting the analog gain, with some caveats (like I'm not sure 1 actually means unity gain).
